slsDetectorSoftware/include/sls/Detector.h

```diff
@@ -1131,15 +1131,23 @@ class Detector {
 
     /** [Jungfrau] Advanced
      * //TODO naming
-     * By default, the on-chip gain switching is active during the entire
-     * exposure. This mode disables the on-chip gain switching comparator
-     * automatically after 93.75% of exposure time (only for longer than
-     * 100us).\n
-     * Default is false or this mode disabled(comparator enabled throughout).
-     * true enables mode. 0 disables mode.
+     * By default, the on-chip gain switching is active during the
+     * entire exposure. This mode disables the on-chip gain switching comparator
+     * automatically after 93.75% of exposure time (only for longer than 100us).
+     * The % is for chipv1.0. One can set the duration for chipv1.1 using
+     * setComparatorDisableTime\n Default is false or this mode
+     * disabled(comparator enabled throughout). true enables mode. 0 disables
+     * mode.
      */
     void setAutoCompDisable(bool value, Positions pos = {});
 
+    /** [Jungfrau] */
+    Result<ns> getComparatorDisableTime(Positions pos = {}) const;
```

Comment:
Should we change the name to getCompDisableTime? to be consistent with getAutoCompDisable?

Reply:
done. the other way. change getAutoCompDisable to getAutoComparatorDisable.